src/context/providers/HubbardUContextProvider.js

```diff
@@ -84,15 +112,40 @@ export class HubbardUContextProvider extends mix(JSONSchemaFormDataProvider).wit
                     atomicOrbital: {
                         type: "string",
                         title: "Atomic orbital",
-                        enum: this.orbitalList,
-                        default: defaultHubbardConfig.atomicOrbital,
                     },
                     hubbardUValue: {
                         type: "number",
                         title: "Hubbard U (eV)",
                         default: defaultHubbardConfig.hubbardUValue,
                     },
                 },
+                dependencies: {
+                    atomicSpecies: {
+                        oneOf: this.uniqueElementsWithLabels.map((elementWithLabel) => {
```

Comment:
We should make the code reusable here to avoid repeating below

When handling elementWithLabel we can have a helper function to getLabel or getElementSymbol to make it easier to understand or we can have a class for it.